Drivers/Inc/BatteryBalancing.h

```diff
@@ -0,0 +1,54 @@
+/** BatteryBalancing.h
+ *  functions for battery balancing.
+ */
+#ifndef BATTERY_BALANCING_H
+#define BATTERY_BALANCING_H
+
+
+#include "LTC6811.h"
+#include "Voltage.h"
+
+/**
+ * @brief   Finds lowest voltage amongst the 31 modules. Meant to be called only in Balancing_Balance()
+ * @param   None
+ * @return  the  
+ */
+static uint16_t Balancing_Init(void);
+
+/**
+ * @brief   Loops through all 31 modules, sets discharge bits for any module if its voltage is too high, and clears discharge bits for any modules with voltages that are too low
+ * @param   Minions array of the ICs that the modules are connected to
+ * @return  None
+ */
+void Balancing_Balance(cell_asic Minions[]);
```

Comment:
Is this the only function that should be called by the battery balancing task, or by a main loop? If so, we should only expose this function.